modules/administration-guide/pages/limiting-the-number-of-workspaces-that-a-user-can-keep.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:navtitle: Limiting the number of workspaces that a user can keep
+:description: Limiting the number of workspaces that a user can keep
+:keywords: administration guide, number, workspaces
+:page-aliases: configuring-the-number-of-workspaces-that-a-user-can-create.adoc
+
+[id="limiting-the-number-of-workspaces-that-a-user-can-keep_{context}"]
+= Limiting the number of workspaces that a user can keep 
+
+By default, users can keep an unlimited number of workspaces in the dashboard, but you can limit this number to reduce demand on the cluster.
+
+.Procedure
+
+. Get the name of the {prod-short} namespace. The default is `{prod-namespace}`.
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get checluster --all-namespaces \
+  -o=jsonpath="{.items[*].metadata.namespace}"
+----
+
+. Configure the `CheCluster` Custom Resource. See xref:using-the-cli-to-configure-the-checluster-custom-resource.adoc[].
++
+[source,yaml,subs="+quotes"]
+----
+spec:
+  components:
+    cheServer:
+      extraProperties:
+        CHE_LIMITS_USER_WORKSPACES_COUNT: "__<max_number_of_kept_workspaces_per_user>__" <1>
```

Comment:
you could rename this from `max_number_of_kept_workspaces_per_user` to `max_count_of_defined_workspaces_per_user`

Reply:
Thank you, Nick. Renaming.
```suggestion
        CHE_LIMITS_USER_WORKSPACES_COUNT: "__<max_count_of_kept_workspaces_per_user>__" <1>
```